pkg/apis/batch/v1alpha1/job.go

```diff
@@ -178,7 +178,7 @@ type LifecyclePolicy struct {
 	// The Event recorded by scheduler; the controller takes actions
 	// according to this Event.
 	// +optional
-	Event Event `json:"event,omitempty" protobuf:"bytes,2,opt,name=event"`
+	Event []Event `json:"event,omitempty" protobuf:"bytes,2,opt,name=event"`
```

Comment:
Add a new member; we already publish this CRD; we can not change it.

Reply:
Added a new member EventList